docs/decision-log/media-type-versioning.md

```diff
@@ -5,35 +5,150 @@ parent: Decision Records
 
 ## Status
 
-* Status: `accepted`
+* Status: `committed`
 
 ## Context
 
-The TROLIE project needs a plan for updating the specification after the
-1.0 scope is complete, that is, after the spec is published as 1.0 and clients
-and servers are deployed that are dependent on that version of the spec.
+The primary mandate of the TROLIE project is defined by its
+[Scope](https://github.com/trolie/spec/blob/1.0.0-wip/Scope.md).  Paramount to
```

Comment:
You're going to want to make these links relative so they don't die when you change the branch name.